README.md

```diff
@@ -15,3 +15,189 @@ To store endpoint data:
 
 - MongoDB >= 3.6 as we need dotted key support (e.g. `{ 'key.path': 'xyz' }`)
 - Your MongoDB set up needs to be a replica set
+
+## Setup Mongo Replica Set
+
+### Yarn Mapping Mediator
+
+This set up guide will make use of Docker to run the Mongo cluster and yarn to run the Mapping Mediator. The containers will connect with each other over a Docker Network.
+
+The easiest way to setup the mongo containers is to run these commands in a `docker-compose` script. To do this copy the code below into a file named `docker-compose.yml`
+
+```yaml
+version: '3.3'
+
+networks:
+  mapper-cluster-network:
+
+services:
+  mapper-mongo-1:
+    image: mongo:4.2
+    container_name: mapper-mongo-1
+    post:
+      - "27017:27017"
+    networks:
+      - mapper-cluster-network
+    command:
+      - --replSet
+      - mapper-mongo-set
+
+  mapper-mongo-2:
+    image: mongo:4.2
+    container_name: mapper-mongo-2
+    post:
+      - "27018:27017"
+    networks:
+      - mapper-cluster-network
+    command:
+      - --replSet
+      - mapper-mongo-set
+
+  mapper-mongo-3:
+    image: mongo:4.2
+    container_name: mapper-mongo-3
+    post:
+      - "27019:27017"
+    networks:
+      - mapper-cluster-network
+    command:
+      - --replSet
+      - mapper-mongo-set
+```
+
+Run the script with the following command:
+
+```sh
+docker-compose up -d
+```
+
+Once the containers have started up exec into one of the containers with the following command, `docker exec -it mapper-mongo-1 mongo`
+
+Inside the shell enter the following:
```

Comment:
Maybe we can add a note here to inform the user that the IP being used is the local docker bridged IP. This is generally the same but depending on how you configured docker it might be a different IP address